src/backend/patches/README.md

```diff
@@ -4,10 +4,9 @@ These patches use [patch-package](https://github.com/ds300/patch-package) to upd
 
 ## `@mapeo/core`
 
-### [Fix quickbit dynamic require](./@mapeo+core+9.0.0-alpha.5+001+fix-quickbit-dynamic-require.patch)
+### [Fix quickbit dynamic require](@mapeo+core+9.0.0-alpha.6.patch)
```

Comment:
nit: I prefer to use `patch-package`'s [`--append`](https://github.com/ds300/patch-package/?tab=readme-ov-file#creating-multiple-patches-for-the-same-package) feature for deps where we may have multiple unrelated patches, as it helps for documentation and clarity of the changes made

Reply:
Done in 272014c159e7b9fa1620c239332acc787929e97e.